src/database/migrations/1620171366267-CreateSongs.ts

```diff
@@ -8,14 +8,14 @@ export class CreateSongs1620171366267 implements MigrationInterface {
       columns: [
         {
           name: 'id',
-          type: 'integer',
+          type: 'int',
           isPrimary: true,
           isGenerated: true,
           generationStrategy: 'increment',
         },
         {
           name: 'playlist_id',
-          type: 'varchar',
+          type: 'integer',
```

Comment:
```suggestion
          type: 'varchar',
```

deixar a coluna do tipo primitivo `string` porque o `id` do servidor pode começar com `0's`, e podem ser ignorados.

Reply:
Opa, eu coloquei o tipo como integer pois o typeorm acusava que ele tinha que ter o mesmo tipo do id da tabela playlist, então o ideal é mudar o tipo de id de playlist para varchar também? como se lida com a geração desse id como varchar?

Reply:
pode deixar como `integer` mesmo, quem precisa ser `varchar` é o `guild_id`